layouts/partials/fragments/nav.html

```diff
@@ -81,6 +81,13 @@
               </a>
             </li>
           {{- end -}}
+        {{- end -}}
+        {{- with ($page_scratch.Get "memberspace") }}
```

Comment:
Adding a vendor specific fragment into another fragments code seems problematic.
We should inject it from a memberspace fragment or at least make this injection more generic.
As it currently stands this seems too bundled.